jboss-eap-clustered-rhel-VMSS/README.md

```diff
@@ -43,19 +43,19 @@ This uses RHEL 8.0 image which is a PAY AS YOU GO image and doesn't require the
 
 ## Prerequisites
 
-1. Azure Subscription with specified payment method (RHEL 8 is a market place product and requires payment method to be specified in Azure Subscription)
+1. Azure Subscription with specified payment method (RHEL 8.0 is a market place product and requires payment method to be specified in Azure Subscription)
```

Comment:
For all the other template we have just mentioned as RHEL 8, please let us know whether to go with that or else I can edit in all the templates to RHEL 8.0 (wherever applicable)